conan/tools/cmake/toolchain/blocks.py

```diff
@@ -736,6 +744,11 @@ def context(self):
 
         system_name, system_version, system_processor = self._get_cross_build()
 
+        cmake_make_program = self._conanfile.conf.get("tools.gnu:make_program",
```

Comment:
Maybe the ``make_program`` will only be set in mingw profiles, nobody will define it if not necessary, no need to check the ``is_mingw()`` and no need to make it none? Maybe the variable is also useful for other compilers like ``qcc`` (qnx)?

Should it have a default ``make`` value or this is only necessary when make is not in the path?